src/tests/banners/SubscriptionsBannerTest.ts

```diff
@@ -0,0 +1,17 @@
+import { BannerPageTracking, BannerTargeting, BannerTest } from '../../components/BannerTypes';
```

Comment:
We prefer absolute paths in support-frontend, not sure if that is the convention here but it seems neater than these  complicated relative ones.

Reply:
I had a look at this and it seems as though it may require a plugin. I might create a separate card.